src/lib/card/card.scss

```diff
@@ -62,11 +62,10 @@ $mat-card-header-size: 40px !default;
 }
 
 .mat-card-footer {
-  position: absolute;
-  width: 100%;
-  min-height: 5px;
-  bottom: 0;
-  left: 0;
+  $margin: -$mat-card-default-padding;
```

Comment:
Could you add comments that explain the negative margins? I find those to usually be hard to grok when coming back to the styles later.